### DISCUSSION TITLE
ScriptReference/Transform.SetSiblingIndex

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#301] [Notes]
# ScriptReference/Transform.SetSiblingIndex

https://docs.unity3d.com/ScriptReference/Transform.SetSiblingIndex.html

**Comment by docscomments** [comment]
**Comment by `@tenpn`:**  

Range is `0 <= index < transform.parent.childCount`.

**Reply by Chrispykins** [reply]
Any index given outside this range will cause the transform to be pushed to the end of the sibling list.
